### ISSUE TITLE
png to texb

### ISSUE THREAD

**Issue by QuickGlare** [MikuAuahDark/Itsudemo#3]
Hi, How do i convert a .png file in a .texb file?

**Comment by MikuAuahDark** [comment]
2 steps
1. Generate blank TEXB with Future of the Flowers.
2. Replace the texture with Itsudemo.
24 Sep 2016 01.25, "Pierpaolo" notifications@github.com menulis:

**Comment by QuickGlare** [comment]
Future of the Flowers? Where i can download it?

**Comment by MikuAuahDark** [comment]
Yep. It's used to generate empty TEXB with specificied resolution. It's
available in my GitHub as Mirai No Hana.
24 Sep 2016 01.48, "Pierpaolo" notifications@github.com menulis:

**Comment by QuickGlare** [comment]
Sorry, but i don't see Mirai No Hana in your github, i see only SIF_Win32 NPPS HonokaMiku Itsudemo SharpMiku HitotsuNoHikari sautracks 2dCustomNumberLua hatedelay SIFTools

**Comment by MikuAuahDark** [comment]
(Facepalm). It's currently propieraty application, but I'll release it as open source once I got it cleaned up.

Anyway, if you want to replace image in TEXB, use `Itsudemo -r <timg name>:<png path> input.texb`

**Comment by QuickGlare** [comment]
Ok
